### ISSUE TITLE
Compatibility with USB Vision cameras?

### ISSUE THREAD

**Issue by JulienMaille** [roboception/rc_genicam_api#32]
I see this ships with Baumer transport layer for gige but also usb cameras.
Is there anything missing so that it could also work with USB Vision compliant devices?
Thanks in advance.

PS: I'm also trying to understand what's missing in EMVA Reference Implementation for users that want to adress a camera without using proprietary SDKs, if anyone can point me to a wiki/manual explaining that, it would be awesome!

**Comment by heikohimu** [comment]
The reason for implementing the rc_genicam_api was to fill the gap between the reference implementation and the transport layers.

Both transport layers work under Linux and at least the GigE transport layer works under Windows. I recently found out that the USB transport layer does not work under Windows. It works after installing the Baumer SDK (BGAPI). It seems that there is something missing, but I did not find the time to check what it is. For us, only GigE is relevant. USB was packaged as I thought it is useful.
